mantaray/received.py

```diff
@@ -312,12 +332,27 @@ def _handle_cap(server_view: views.ServerView, args: list[str]) -> None:
     if subcommand == "ACK":
         acknowledged = set(args[-1].split())
         if "sasl" in acknowledged:
+            server_view.core.cap_list.add("sasl")
             server_view.core.send("AUTHENTICATE PLAIN")
+        if "away-notify" in acknowledged:
+            server_view.core.cap_list.add("away-notify")
     elif subcommand == "NAK":
         rejected = set(args[-1].split())
         if "sasl" in rejected:
             # TODO: this good?
             raise ValueError("The server does not support SASL.")
+        if "away-notify" in rejected:
+            raise ValueError("The server does not support away-notify.")
```

Comment:
Mantaray should be able to work with servers that don't support away-notify. It shouldn't be an error. It just means you won't get colors in the userlist, and that's all it does.